roles/kustomize_deploy/molecule/default/prepare.yml

```diff
@@ -0,0 +1,70 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Prepare
+  hosts: all
+  vars_files:
+    - ../../defaults/main.yml
+  vars:
+    _install_yamls_path: >-
+      {{
+        cifmw_installyamls_repos |
+        default(
+          '~/src/github.com/openstack-k8s-operators/install_yamls',
+          true
+        )
+      }}
+  pre_tasks:
+    - name: Ensure CRC is started
+      async: 1800
+      poll: 0
+      register: _crc_start
+      ansible.builtin.command:
+        cmd: crc start
+  roles:
+    - role: test_deps
```

Comment:
it would be nice if these were also skipable

if you are not including this for a specific reason its best not to execute it

Reply:
I don't think that is needed here, just a leftover from the "template".